### PR TITLE
Added Vue Links

### PR DESCRIPTION
**Pull request by JamesLF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,14 @@
    * [React Documentation](https://reactjs.org/docs/getting-started.html)
    * [React Tutorial](https://reactjs.org/tutorial/tutorial.html)
 
+* Vue
+   * [Vue Documentation](https://vuejs.org/v2/guide/)
+   * [Vee Validate](https://baianat.github.io/vee-validate/) - Client-side Validation
```
Comment (dextermb) [inline README.md]: Phew. Thank god _vee_ is in the name of the library

Comment (DivineOmega) [reply README.md]: I had to double-check that too. :+1:

Comment (wheatleyjj) [reply README.md]: I saw this and panicked! 😄

Comment (AlexCatch) [reply README.md]: What's Vue?

**Pull request merged**